Source/PluginProcessor.cpp

```diff
@@ -205,11 +274,27 @@ void SoomplerAudioProcessor::setTransportStateListener(TransportStateListener* l
     this->transportStateListener = listener;
 }
 
-double SoomplerAudioProcessor::getCurrentAudioPosition() const
+double SoomplerAudioProcessor::getCurrentAudioPosition()
 {
+    if (synth.getVoice(0)->isVoiceActive()) {

```

Comment:
what if getVoice(0) is null ? SEGFAULT

Reply:
yes, this error will indicate error with synth voices in every application run

Reply:
overchecking can spoil code in my opinion. There is no expected reason voice can be null